aws/credentials/processcreds/provider.go

```diff
@@ -0,0 +1,353 @@
+/*
+Package processcreds is a credential Provider to retrieve `credential_process`
+credentials.
+
+**Warning:**
```

Comment:
@xibz I copied the same warning from CLI here. Let me know if it's needed elsewhere.